src/ai/susi/server/api/cms/SkillMetricsDataService.java

```diff
@@ -112,6 +113,10 @@ public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Aut
 
                     jsonArray.put(skillMetadata);
                     skillObject.put(skill_name, skillMetadata);
+
+                    if(SusiSkill.isStaffPick(model_name, temp_group_name, language_name, skill_name)) {
```

Comment:
This block of code also needs to be written when group name is not `All`

Reply:
Please add the lines.

Reply:
Done. Please review. Thanks :-)